deployment/docker/uwsgi.conf

```diff
@@ -19,7 +19,7 @@ logger = file:/var/log/uwsgi-errors.log
 #gid = 1000
 memory-report = true
 harakiri = 20
-plugins = python34
+plugins = python
```

Comment:
WHat is the correct setting for this? I think we had problems with both python and python34